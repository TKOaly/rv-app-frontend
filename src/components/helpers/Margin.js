import React from 'react';

const Margin = (props) => (
    <div
        className="margin"
        style={{
            margin: props.margin,
            display: props.inlineBlock ? 'inline-block' : 'block'
        }}
    >
        {props.children}
    </div>
);

export default Margin;
