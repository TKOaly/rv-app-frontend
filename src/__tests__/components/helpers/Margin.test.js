import { shallow } from 'enzyme';
import Margin from '../../../components/helpers/Margin';
import React from 'react';

describe.only('<Margin />', () => {
    it('should match snapshot', () => {
        const cmpnt = shallow(<Margin margin={20}>HelloWorld</Margin>);
        expect(cmpnt).toMatchSnapshot();
    });
});
