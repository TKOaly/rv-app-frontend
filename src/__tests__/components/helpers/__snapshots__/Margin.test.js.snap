// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Margin /> should match snapshot 1`] = `ShallowWrapper {}`;
